sandbox/basic.css

```diff
@@ -224,4 +224,4 @@ Success {
 
 .horizontal {
   layout: horizontal
-}
\ No newline at end of file
+}
```

Comment:
just briefly touched this file for my tests, and my IDE added this missing line return at the end of it :slightly_smiling_face:

Reply:
The pre-commit hooks add missing newlines too

Reply:
Ah, true! Well that's many tools trying hard to add this newline then, we should be safe :smile: 

(just checked my own PyCharm settings, and I do have this option enabled on top of pre-commit :-)
![Screenshot from 2022-05-10 11-54-34](https://user-images.githubusercontent.com/722388/167613103-793f2bff-601c-4415-9e6b-f873fd055f6a.png)